app/components/Story.js

```diff
@@ -45,31 +46,36 @@ class Story extends Component {
 
   _renderPage(data) {
     return <StoryCard
-              topText={data.topText} 
-              bottomText={data.bottomText} 
-              isSplit={this.state.isSplit} 
+              topText={data.topText}
+              bottomText={data.bottomText}
+              isSplit={this.state.isSplit}
             />;
   }
 
   render() {
     return (
       <View style={styles.container}>
-        <TouchableHighlight
-          style={styles.button}
-          onPress={this.onClickBack}
-        >
-          <Text>Back</Text>
-        </TouchableHighlight>
         <ViewPager
           dataSource={this.state.data}
           renderPage={this._renderPage}
         />
-        <TouchableHighlight
-          style={styles.toggle}
-          onPress={this.onClickToggle}
-        >
-          <Text>Toggle View</Text>
-        </TouchableHighlight>
+        <View style={styles.bar}>
```

Comment:
There's now a StoryControlPane.js file that has this toolbar (currently with emoji icons), so these changes will need to be incorporated there.